framework/generator/generator_test.go

```diff
@@ -180,7 +180,7 @@ func TestSyntaxError(t *testing.T) {
 	cli := testcli.New(td.Directory())
 	res, err := cli.Run(ctx, "build", "--embed=false")
 	is.True(err != nil)
-	is.In(err.Error(), `exit status 2`)
+	is.In([]string{`exit status 1`, `exit status 2`}, err.Error()) // The exit status code changed in Go v1.20 from "2" to "1".
```

Comment:
Did you see this documented somewhere online? 

I had a look at the release notes, but didn't spot anything. Also grepped the code, but no `os.Exit(2)`

Reply:
Sadly no, this "fix" is only based on empirical analysis of Go-version-based behavior :see_no_evil: 

Would like to have this backed by objective facts as well, but could not yet find any. Did also seek through the release notes as well, but could not find any changes regarding this.

Where exactly did you grep for `os.Exit(2)`? 
Quickly grepped through go1.19.7 and go1.20.2 sources which at least gave me a different count sfor it ;) (I would assume  Go <= 1.19 to return with `2` and Go >= 1.20 to return with `1`)